### PR TITLE
Updating to Ember 2.3; Removed utils/can

### PR DESCRIPTION
**Pull request by mileszim**
Hello, thank you for the great ember addon! I have updated the entire addon to Ember 2.3 and ember-cli 2.3.0 code, as well as the tests. Additionally, the implementation of utils/can no longer functions or makes sense with the container/getOwner changes in Ember 2, in addition to no real documentation on it, so I have removed it.

I'm happy to discuss & make changes as you see fit!

### CODE REVIEW

File: addon/computed.js [hunk 1]
```diff
@@ -10,13 +7,12 @@ export default {
...
-      const ability   = container.lookup("ability:"+type);
+      const ability = Ember.getOwner(this).lookup("ability:" + type);
 
-      Ember.assert("No ability class found for "+type, ability);
+      Ember.assert("No ability class found for " + type, ability);
 
-      const resource = get(this, resourceName);
-      set(ability, "model", resource);
+      const resource = Ember.get(this, resourceName);
+      Ember.set(ability, "model", resource);
```
Comment (blimmer) [inline addon/computed.js]: My prefered style for these is to use destructure these like this:

```javascript

const { get, set } = Ember;

get(this, resourceName);
...
```

Comment (mileszim) [inline addon/computed.js]: Added that in. Any other comments?

**Comment by Exelord** [comment]
Please merge :) :+1:

File: addon/services/can.js [hunk 2]
```diff
@@ -8,9 +8,9 @@ export default Ember.Service.extend({
 
   build(abilityString, resource, properties) {
     const names   = this.parse(abilityString);
-    const ability = this.container.lookup("ability:"+names.abilityName);
+    const ability = Ember.getOwner(this).lookup("ability:" + names.abilityName);
 
-    Ember.assert("No ability type found for "+names.abilityName, ability);
+    Ember.assert("No ability type found for " + names.abilityName, ability);
```
Comment (blimmer) [inline addon/services/can.js]: I'd personally prefer string concatenation here for clarity.

**Comment by blimmer** [comment]
This looks good to me. @rlivsey only thing I'm not 100% on is the removal of `util/can` - assuming it's OK to ditch, but wanted to confirm with you before merging.

**Comment by floqqi** [comment]
Any progress on this one? :)

**Comment by rlivsey** [comment]
Sorry, it's been a crazy week and I've been a bit neglectful. Getting ready to launch our new app so I've been a bit snowed under. I'm taking a look now.

**Comment by floqqi** [comment]
No problem, thanks anyway! Ping me if you need some help :)

**Comment by rlivsey** [comment]
Released as `0.8.0`, thanks again

**Comment by floqqi** [comment]
Just came from lunch and the release was there. Awesome, thanks! :+1:

**Pull request merged**